lib/shared-mongo-connections.js

```diff
@@ -0,0 +1,76 @@
+var mongoUrlParser = require('mongodb-uri');
+var async = require('async');
+var MongoClient = require('mongodb').MongoClient;
+var mongoose = require('mongoose');
+
+const MONGODB_DEFAULT_POOL_SIZE = 20;
+
+/**
+ * Get the url to connect to the admin mongodb specified in the given `config` object.
+ * @param {string} mongoConnectionString a mongodb url that should at least contain the mongodb hosts
+ * @param {object} mongoConf the mongodb configuration
+ * @param {string} mongoConf.admin_auth.user the admin username
+ * @param {string} mongoConf.admin_auth.pass the amdin user password
+ * @param {int} mongoConf.poolSize the poolSize of the mongodb connection. Default to 20.
+ * @return {string} the mongodb admin connection url 
+ * 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)